test/cross-browser-testing/rollup.CBT.config.js

```diff
@@ -8,7 +8,7 @@ import json from '@rollup/plugin-json';
 
 export default [
     {
-        input: 'test/src/tests-main.js',
+        input: 'test/src/_test.index.ts',
```

Comment:
As per our conversation this morning, I thought we were just going to call this `index.ts`?